client/rest/handlers.go

```diff
@@ -0,0 +1,284 @@
+		writeError(w, err)
+		return
+	}
+
+	query := mux.Vars(r)
+	name := query["name"]
+	if name != uReq.Name {
+		writeError(w, errNonMatchingPathAndJSONKeyNames)
+		return
+	}
+
+	if err := k.manager.Update(uReq.Name, uReq.OldPass, uReq.NewPass); err != nil {
+		writeError(w, err)
+		return
+	}
+
+	key, err := k.manager.Get(uReq.Name)
+	if err != nil {
+		writeError(w, err)
+		return
+	}
+	writeSuccess(w, &key)
+}
+
+func (k *Keys) DeleteKey(w http.ResponseWriter, r *http.Request) {
+	dReq := new(DeleteKeyRequest)
+	if err := parseRequestJSON(r, dReq); err != nil {
+		writeError(w, err)
+		return
+	}
+
+	query := mux.Vars(r)
+	name := query["name"]
+	if name != dReq.Name {
+		writeError(w, errNonMatchingPathAndJSONKeyNames)
+		return
+	}
+
+	if err := k.manager.Delete(dReq.Name, dReq.Passphrase); err != nil {
+		writeError(w, err)
+		return
+	}
+
+	resp := &ErrorResponse{Success: true}
+	writeSuccess(w, resp)
+}
+
+func (k *Keys) Register(r *mux.Router) {
+	r.HandleFunc("/keys", k.GenerateKey).Methods("POST")
+	r.HandleFunc("/keys", k.ListKeys).Methods("GET")
+	r.HandleFunc("/keys/{name}", k.GetKey).Methods("GET")
+	r.HandleFunc("/keys/{name}", k.UpdateKey).Methods("POST", "PUT")
+	r.HandleFunc("/keys/{name}", k.DeleteKey).Methods("DELETE")
+}
+
+type Context struct {
+	Keys *Keys
+}
+
+func (ctx *Context) RegisterHandlers(r *mux.Router) error {
+	ctx.Keys.Register(r)
+	r.HandleFunc("/build/send", doSend).Methods("POST")
+	r.HandleFunc("/sign", doSign).Methods("POST")
+	r.HandleFunc("/tx", doPostTx).Methods("POST")
+	r.HandleFunc("/query/account/{signature}", doAccountQuery).Methods("GET")
+
+	return nil
+}
+
+func extractAddress(signature string) (address string, err *ErrorResponse) {
+	// Expecting the signature of the form:
+	//  sig:<ADDRESS>
+	splits := strings.Split(signature, ":")
+	if len(splits) < 2 {
+		return "", &ErrorResponse{
+			Error: `expecting the signature of the form "sig:<ADDRESS>"`,
+			Code:  406,
+		}
+	}
+	if splits[0] != "sigs" {
+		return "", &ErrorResponse{
+			Error: `expecting the signature of the form "sig:<ADDRESS>"`,
+			Code:  406,
+		}
+	}
+	return splits[1], nil
+}
+
+func doAccountQuery(w http.ResponseWriter, r *http.Request) {
+	query := mux.Vars(r)
+	signature := query["signature"]
+	address, errResp := extractAddress(signature)
+	if errResp != nil {
+		writeCode(w, errResp, errResp.Code)
+		return
+	}
+	actor, err := commands.ParseActor(address)
+	if err != nil {
+		writeError(w, err)
+		return
+	}
+	actor = coin.ChainAddr(actor)
+	key := stack.PrefixedKey(coin.NameCoin, actor.Bytes())
+	account := new(coin.Account)
+	proof, err := proofs.GetAndParseAppProof(key, account)
+	if lightclient.IsNoDataErr(err) {
+		err := fmt.Errorf("account bytes are empty for address: %q", address)
+		writeError(w, err)
+		return
+	} else if err != nil {
+		writeError(w, err)
+		return
+	}
+
+	if err := proofs.OutputProof(account, proof.BlockHeight()); err != nil {
+		writeError(w, err)
+		return
+	}
+	writeSuccess(w, account)
```

Comment:
If you look at OutputProof, it just adds an envelope around the data to add height info to the json.  You can do it here before writeSuccess.